Google.Api.Gax.Grpc/ServiceMetadata.cs

```diff
@@ -35,8 +35,17 @@ public sealed class ServiceMetadata
         /// <summary>
         /// The default endpoint for the service. This may be null, if a service has no default endpoint.
         /// </summary>
+        /// <remarks>
+        /// The default endpoint is an endpoint in the default universe domain.
+        /// </remarks>
         public string DefaultEndpoint { get; }
 
+        /// <summary>
+        /// The template to build and endpoint for the service taking into account a custom universe domain.
+        /// May be null, in which case no universe domain dependant endpoint may be built for the service.
```

Comment:
dependant => dependent

It might also be worth giving an example in remarks - "template" sounds like there's a placeholder or similar, and we can't really tell what that is here. (Given the above code, it looks like it's going to be something like `storage.{0}` or similar, but an example would be good.)